src/Runtime/ExecutionSession.hpp

```diff
@@ -41,10 +41,17 @@ class ExecutionSession {
   // tensor lists.
   OMTensorList *run(OMTensorList *input);
 
+  // Set entry point for this session.
```

Comment:
I would add that this call is required before calling run/input/output signatures when default entry point is not selected. To that effect, I would move this def above the definition of run, so that the sequence is clearer.

Reply:
Yes, I changed the order.